code/game/turfs/simulated/wall/reinf_walls.dm

```diff
@@ -15,6 +15,9 @@
 
 /turf/closed/wall/r_wall/examine(mob/user)
 	..()
+	deconstruction_hints_rwall(user)
+
+/turf/closed/wall/r_wall/proc/deconstruction_hints_rwall(mob/user)
 	switch(d_state)
```

Comment:
I tried to move the "deconstruction_hints_rwall"-stuff over to walls.dm's deconstruction_hints, so all deconstruction hints would be under one proc, but I couldn't figure out a way to get it recognize rwall's d_state.

Reply:
simple; remove the `/proc` and `_rwall` sections from line 20 to leave `/turf/closed/wall/r_wall/deconstruction_hints(mob/user)`